tests/lax_numpy_operators_test.py

```diff
@@ -635,7 +635,7 @@ def testBitwiseCount(self, shape, dtype):
       for dtypes in itertools.product(
         *(_valid_dtypes_for_shape(s, int_dtypes_no_uint64) for s in shapes))
     ],
-    op=[jnp.left_shift, jnp.bitwise_left_shift, jnp.right_shift, jnp.bitwise_right_shift],
```

Comment:
We should continue to test these, because they're public APIs and we should make sure they work as documented.

Reply:
Hmm... I removed them for a reason, but I tested it locally and they seem to be working again, let's see what CI/CD says, but yeah added it back.